js/src/jit-test/tests/ion/pow-base-power-of-two.js

```diff
@@ -1,3 +1,5 @@
+// |jit-test| skip-if: getBuildConfiguration("pbl") && getBuildConfiguration("debug")
```

Comment:
Same for this one, a comment explaining why we skip would be useful to have

Reply:
Done!